resources/testharness.js

```diff
@@ -680,6 +681,26 @@ policies and contribution forms [3].
         });
     }
 
+    /**
+     * See RFC 75:
+     * https://github.com/web-platform-tests/rfcs/blob/master/rfcs/async_promise_test.md
```

Comment:
Nit: I'd put this at the bottom of the comment block, if at all.

Reply:
Done